pycognito/aws_srp.py

```diff
@@ -202,18 +202,41 @@ def get_secret_hash(username, client_id, client_secret):
         hmac_obj = hmac.new(bytearray(client_secret, "utf-8"), message, hashlib.sha256)
         return base64.standard_b64encode(hmac_obj.digest()).decode("utf-8")
 
+    @staticmethod
+    def get_cognito_formatted_timestamp(input_datetime):
+        weekday_names = ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"]
+        month_names = [
```

Comment:
Let's put that on top of the file as constant kinda `WEEKDAY_NAMES` and `MONTH_NAMES`

Reply:
Sure, fixed.